routing/payment_session.go

```diff
@@ -290,6 +290,12 @@ func (p *paymentSession) RequestRoute(maxAmt, feeLimit lnwire.MilliSatoshi,
 				return nil, errNoPathFound
 			}
 
+			if p.payment.DestFeatures == nil {
+				p.log.Debug("not splitting because " +
```

Comment:
nit: Log messages usually are capitalized, only error messages are left lower case.

Reply:
Totally forgot😂 one day I will change the rest log messages too haha